bin-src/main.ts

```diff
@@ -1,5 +1,7 @@
 import { run } from '../node-src';
 
 export async function main(argv: string[]) {
-  return run({ argv });
+  const { code } = await run({ argv });
+
+  process.exit(code);
```

Comment:
This would normally not be necessary, something is keeping node open, you should probably investigate what. This `.exit()` method can prevent errors from getting fully printed..

https://www.npmjs.com/package/why-is-node-running

Maybe we add a ticket to investigate this, or add a TODO comment?